src/lib/options.ts

```diff
@@ -9,7 +9,7 @@ export interface CustomOptions {
   maxGapSize?: number;
   maxHashCount?: number;
   maxHashPercent?: number;
-  maxMatches?: number;
+  maxMatches?: number; // still needed?
```

Comment:
```suggestion
  maxMatches?: number;
```

The `maxMatches` option is still used [here](https://github.com/dodona-edu/dolos/blob/1d6061497c34918555f7e4b02b54f9a53c3bb76a/src/lib/summary.ts#L89), or what do you mean?

Reply:
I must have overlooked that reference! I removed the comments.